test-backend.sh

```diff
@@ -14,6 +14,9 @@ set -e
 
 export GOBIN=${PWD}/bin:${GOBIN}
 
+# Use deps from vendor dir.
+export GOFLAGS=-mod=vendor
```

Comment:
I would introduce quote here:

```
GOFLAGS="-mod=vendor"
```

also apply this to `build-backend.sh` as well.

Reply:
Fixed https://github.com/openshift/console/pull/4007/commits/9fcc599bb393bf290332ea071d88321d5f28e6d4